tests/TopDownProteomics.Tests/TopDownProteomics.Tests.csproj

```diff
@@ -18,6 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="TestData\topPicTestMods.txt" />
```

Comment:
the changes to this file can removed. VS likes to add this for some reason even though it isn't needed.